src/form/Checkbox/Checkbox.tsx

```diff
@@ -20,6 +21,11 @@ export interface CheckboxProps {
    */
   label?: string;
 
+  /**
+   * Label position of checkbox.
+   */
+  labelPosition?: 'left' | 'right';
```

Comment:
Use start/end for more rtl friendly naming

```suggestion
  labelPosition?: 'start' | 'end';
```